EMML/1-1000/EMML152.xml

```diff
@@ -95,7 +95,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
       <revisionDesc>
          <change who="PL" when="2017-11-30">Created minimal record with link from data sent from Daniel Gullo, HMML
                             project.</change>
-                            <change who="HA" when="2022-04-06"dimensions<change>
+                            <change who="HA" when="2022-04-06">dimensions<change>
```

Comment:
dear Hagos,
the close tag should always look like
`</thenameofthetag>`, with a slash before the name of the tag
so in your case
`<change who="HA" when="2022-04-06">dimensions</change>`